lesson3/task2,3.js

```diff
@@ -0,0 +1,16 @@
+// 2. С этого урока начинаем работать с функционалом интернет-магазина. 
+// Предположим, есть сущность корзины. 
+// Нужно реализовать функционал подсчета стоимости корзины в зависимости от находящихся в ней товаров.
+// 3. Товары в корзине хранятся в массиве. Задачи:
+// a.Организовать такой массив для хранения товаров в корзине;
+// b. Организовать функцию countBasketPrice, которая будет считать стоимость корзины.
+
+let bascket1 = new Array(1,1,1,1,1);
```

Comment:
Не забывай ставить пробелы
```suggestion
let bascket1 = new Array(1, 1, 1, 1, 1);
```